src/NewFamilyMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import './NewFamilyMember.scss';
+
+enum DocumentType {
+  type,
+  dni,
+  nie,
+  pasaporte
+};
+
+function NewFamilyMember() {
+  const [name, setName] = useState<string>('');
+  const [surname, setSurname] = useState<string>('');
+  const [date, setDate] = useState<string>('');
+  const [documentType, setDocumentType] = useState<DocumentType>(DocumentType.type);
```

Comment:
Not required, but instead of having each property separated, you can have a single prop called `newFamilyMember` that will be an object containing those props